src/pal/tools/gen-buildsys-gcc.sh

```diff
@@ -37,21 +37,30 @@ elif command -v "${gcc_prefix}gcc$2$3" > /dev/null
 elif command -v "${gcc_prefix}gcc-$2$3" > /dev/null
     then
         desired_gcc_version="-$2$3"
-elif command -v ${gcc_prefix}gcc > /dev/null
+elif command -v "${gcc_prefix}gcc" > /dev/null
     then
         desired_gcc_version=
 else
-    echo "Unable to find $gcc_prefixgcc Compiler"
+    echo "Unable to find ${gcc_prefix}gcc Compiler"
     exit 1
 fi
 
-if [ -z "$CC" ]; then
-  export CC="$(command -v ${gcc_prefix}gcc$desired_gcc_version)"
+if [ -z "$CLR_CC" ]; then
+    CC="$(command -v "${gcc_prefix}"gcc"$desired_gcc_version")"
```

Comment:
please remove the quotes around gcc

Reply:
Removed. Thanks.
I also set up the cross environment and diff'd with master that the variables passed to cmake in configurecompiler.cmake files: `execute_process(COMMAND "${CMAKE_COMMAND}" "-E" "environment")` with `./build.sh arm64 checked cross gcc`. There are no diffs and the detection says:

> Detected Linux ARM64

Reply:
The only difference in build instruction is `export CC` and `export CXX` has became `export CLR_CC` and `export CXX` respectively; the rest of the steps are same.